src/UserALEWebExtension/background.js

```diff
@@ -41,6 +42,14 @@ const defaultConfig = {
 var urlWhitelist;
 
 function updateConfig(config) {
+  if ((config.toolName || "") === "") {
```

Comment:
in JS you can simply do:
`if (!config.toolName)`

both undefined and "" are "falsy" values.